src/controllers/albumController.js

```diff
@@ -73,9 +143,18 @@ const getAlbumWithAllUsersWhoPurchasedIt = async (req, res) => {
     album: usersThatPurchasedThisAlbum,
     purchasingUsersCount: usersThatPurchasedThisAlbum.purchasedByUsers.length,
   });
+  /*
+     #swagger.summary = 'Show all users that purchased this particular album'
+
+     #swagger.responses[200] = {
+				description: 'Album with users who purchased it was fetched  successfully ',
+				schema: { albums: [{ $ref: '#/definitions/AlbumWithUsers' }], purchasingUsersCount: 1 }
```

Comment:
I made a request to **`http://localhost:8000/api/v1/albums/64d2a94c793389a43fc5a8d6/listOfUsersWhoPurchasedThisAlbum`**

and I got:

```
{
  "album": {
    "_id": "64d2a94c793389a43fc5a8d6",
    "artistName": "F",
    "albumName": "AUSTIN",
    "price": 0,
    "image": "https://i.scdn.co/image/ab67616d0000b27371cae34ad5a39bdab78af13e",
    "releaseDate": "2023-07-28T00:00:00.000Z",
    "spotifyUrl": "https://api.spotify.com/v1/albums/6r1lh7fHMB499vGKtIyJLy",
    "averageRating": 0,
    "numOfReviews": 0,
    "__v": 0,
    "createdAt": "2023-08-08T20:45:00.942Z",
    "updatedAt": "2023-08-17T10:10:42.861Z",
    "purchasedByUsers": [
      {
        "_id": "64ef50c8c5551074444547bc",
        "album": "64d2a94c793389a43fc5a8d6",
        "user": {
          "_id": "64d6ca92a15d2e18ab96a2a3",
          "name": "Akos123123",
          "email": "akos123@example.com",
          "role": "user",
          "__v": 0,
          "username": "akos92",
          "id": "64d6ca92a15d2e18ab96a2a3"
        },
        "__v": 0
      }
    ],
    "id": "64d2a94c793389a43fc5a8d6"
  },
  "purchasingUsersCount": 1
}
```

It returns a single album (the one we passed in the id for) and that album can have an array of users. So our response schema here should match:

```suggestion
				schema: { album: { $ref: '#/definitions/AlbumWithUsers' }, purchasingUsersCount: 1 }
```